src/RamRelation.h

```diff
@@ -32,29 +32,46 @@ namespace souffle {
 
 /**
  * A RAM Relation in the RAM intermediate representation.
+ * TODO: Make RamRelation a sub-class of RAM node.
+ * TODO: Tidy-up interface and attributes
  */
 class RamRelation {
+protected:
+    /** Name of relation */
     std::string name;
+
+    /** Arity, i.e., number of attributes */
     unsigned arity;
+
+    /** Name of attributes */
     std::vector<std::string> attributeNames;
+
+    /** Type of attributes */
     std::vector<std::string> attributeTypeQualifiers;
+
+    /** TODO: legacy, i.e., duplicated information */
```

Comment:
TODOs should be in the form` // TODO (#issueid): text` or `// TODO (userid): text`

Reply:
Done.